### PR TITLE
Update kubeflow/kubeflow manifests from v1.9.0-rc.0

### PR DESCRIPTION
**Pull request by kimwnasptd**
This PR updates the manifests from `kubeflow/kubeflow` components in the manifests for 1.9. 

The manifests are copied from https://github.com/kubeflow/kubeflow/releases/tag/v1.9.0-rc.0

/cc @thesuperzapper @juliusvonkohout @rimolive

### CODE REVIEW

**Comment by juliusvonkohout** [comment]
@kimwnasptd looks good on a first glimpse. The failing test is the only major thing.

**Comment by kimwnasptd** [comment]
It looks like a manifest file was introduced in the manifests repo, but was not part of the source manifests
https://github.com/kubeflow/manifests/blob/v1.9-branch/apps/centraldashboard/upstream/overlays/oauth2-proxy/
https://github.com/kubeflow/kubeflow/tree/v1.9-branch/components/centraldashboard/manifests/overlays

Is this the intended behavior?

**Comment by juliusvonkohout** [comment]
> It looks like a manifest file was introduced in the manifests repo, but was not part of the source manifests https://github.com/kubeflow/manifests/blob/v1.9-branch/apps/centraldashboard/upstream/overlays/oauth2-proxy/ https://github.com/kubeflow/kubeflow/tree/v1.9-branch/components/centraldashboard/manifests/overlays
> 
> Is this the intended behavior?

Either we move

> It looks like a manifest file was introduced in the manifests repo, but was not part of the source manifests https://github.com/kubeflow/manifests/blob/v1.9-branch/apps/centraldashboard/upstream/overlays/oauth2-proxy/ https://github.com/kubeflow/kubeflow/tree/v1.9-branch/components/centraldashboard/manifests/overlays
> 
> Is this the intended behavior?

It is used to patch the logout https://github.com/kubeflow/manifests/blob/v1.9-branch/common/oidc-client/oauth2-proxy/components/central-dashboard/patches/deployment.logout-url.yaml

we need to find a new place for that patch, because it should be maintained within kubeflow/manifests. You can also readd it for now and we tackle that in RC.2.

CC @kromanow94

File: README.md [hunk 1]
```diff
@@ -44,15 +44,15 @@ This repo periodically syncs all official Kubeflow components from their respect
...
-| Profiles + KFAM | apps/profiles/upstream | [v1.8.0](https://github.com/kubeflow/kubeflow/tree/v1.8.0/components/profile-controller/config) |
-| PodDefaults Webhook | apps/admission-webhook/upstream | [v1.8.0](https://github.com/kubeflow/kubeflow/tree/v1.8.0/components/admission-webhook/manifests) |
-| Jupyter Web App | apps/jupyter/jupyter-web-app/upstream | [v1.8.0](https://github.com/kubeflow/kubeflow/tree/v1.8.0/components/crud-web-apps/jupyter/manifests) |
-| Tensorboards Web App | apps/tensorboard/tensorboards-web-app/upstream | [v1.8.0](https://github.com/kubeflow/kubeflow/tree/v1.8.0/components/crud-web-apps/tensorboards/manifests) |
-| Volumes Web App | apps/volumes-web-app/upstream | [v1.8.0](https://github.com/kubeflow/kubeflow/tree/v1.8.0/components/crud-web-apps/volumes/manifests) |
+| Notebook Controller | apps/jupyter/notebook-controller/upstream | [v1.9.0-rc.0](https://github.com/kubeflow/kubeflow/tree/v1.9.0-rc.0/components/notebook-controller/config) |
+| PVC Viewer Controller | apps/pvcviewer-roller/upstream | [v1.9.0-rc.0](https://github.com/kubeflow/kubeflow/tree/v1.9.0-rc.0/components/pvcviewer-controller/config) |
```
Comment (DnPlas) [inline README.md:48]: Is this a typo?

```suggestion
| PVC Viewer Controller | apps/pvcviewer-controller/upstream | [v1.9.0-rc.0](https://github.com/kubeflow/kubeflow/tree/v1.9.0-rc.0/components/pvcviewer-controller/config) |
```

Comment (juliusvonkohout) [reply README.md:48]: > Is this a typo?
> 
> ```suggestion
> | PVC Viewer Controller | apps/pvcviewer-controller/upstream | [v1.9.0-rc.0](https://github.com/kubeflow/kubeflow/tree/v1.9.0-rc.0/components/pvcviewer-controller/config) |
> ```

Yes I think so, but it has been there before the PR.

Comment (juliusvonkohout) [reply README.md:48]: Yes let's fix it

**Comment by rimolive** [comment]
The error is in an overlay for oauth2 for central dashboard. I thought this was tested in kubeflow/kubeflow repo before, wasn't it?

**Comment by juliusvonkohout** [comment]
> The error is in an overlay for oauth2 for central dashboard. I thought this was tested in kubeflow/kubeflow repo before, wasn't it?
> 
> EDIT: I got the whole thread now. What was the final decision about it? It looks like a simple change, so what's the ETA to get this done?

I am in favour of just re-adding it for now and fixing it in RC.2. RC.2 can be used for that and upgrading istio and the sometimes slow oauth2-proxy installation. Let's resynchronize on Monday for that.

**Comment by juliusvonkohout** [comment]
@kimwnasptd Lets close this. It is syncing into the wrong branch. We need to sync to master first and then from master into 1.9.
Otherwise we end up in synchronization hell ;-)

@rimolive Can you run the script to create a new PR against the master branch and undelete /manifests/manifests/apps/centraldashboard/upstream/overlays/oauth2-proxy.

Then we need to synchronize the 1.9 branch to master and you can tag RC.1 this week with https://github.com/kubeflow/manifests/pull/2739

**Comment by rimolive** [comment]
As per Julius suggestion, will close this PR and create a new one with the missing manifest.

/close

**Comment by thesuperzapper** [comment]
@rimolive @juliusvonkohout aside the PR target issue (the PR should obviously be into `master`).

It's probably better if we just move anything that was manually added to the "upstream" folder (this `ouath2-proxy` stuff) into its own folder, so we can sync easily in the future?

**Comment by rimolive** [comment]
@thesuperzapper I agree with that. @juliusvonkohout let's work on this in future releases, or check if it's possible to make that change for 1.9.

**Comment by juliusvonkohout** [comment]
Yes i hope so for 1.9 RC.2

**Pull request closed without merging**